app/validators/questionnaire_validator.py

```diff
@@ -104,3 +105,22 @@ def validate_smart_quotes(self):
                         error_messages.DUMB_QUOTES_FOUND,
                         pointer=translatable_item.pointer,
                     )
+
+    def validate_white_spaces(self):
+        schema_object = SurveySchema(self.schema_element)
+
+        for translatable_item in schema_object.translatable_items:
+            schema_text = translatable_item.value
+            values_to_check = [schema_text]
+
+            if isinstance(schema_text, dict):
+                values_to_check = schema_text.values()
+
+            for text in values_to_check:
+                if text and (
+                    text.startswith(" ") or text.endswith(" ") or "  " in text
+                ):
+                    self.add_error(
```

Comment:
Minor but maybe useful to add the actual text to the error output in addition to the pointer?

Reply:
Error text has now been added as part of error message output.